### PR TITLE
ADDED getIndex() - GET all Projects and all Clients

### PR DESCRIPTION
**Pull request by fallen-ecstasy**
Hey @VighneshManjrekar , I have completed and tested the method on the given data.....Kindly test it from your side too, if any problem let me know

Else Please merge this branch.

Issue #2

### CODE REVIEW

File: controllers/main.controllers.js [hunk 1]
```diff
@@ -5,6 +5,13 @@ const Project = require("../models/Project.model");
 // @route   GET api/
 exports.getIndex = async (req, res, next) => {
   // Fetch all projects and clients
+  try{
+    const projects = await Project.find();
+    const clients = await Client.find();
```
Comment (VighneshManjrekar) [inline controllers/main.controllers.js:10]: include client reference while fetching projects

**Review by VighneshManjrekar — CHANGES_REQUESTED** [review]
Changes are expected to `controllers/main.controllers.js` file only PR should not contain changes of other files or directories

File: controllers/main.controllers.js (hunk 1 above)
Comment (fallen-ecstasy) [reply controllers/main.controllers.js:10]: Okay onto it

**Comment by VighneshManjrekar** [comment]
Closing this PR since no response from contributor

**Pull request closed without merging**